docs/users.rst

```diff
@@ -113,6 +131,59 @@ not
   list of platform names (or a single platform name as a string);
   this restricts the platforms to build for using set difference
 
+compose
+~~~~~~~
+
+This section is used for requesting yum repositories at build time. When this
+section is defined, a compose will be requested by using ODCS.
+
+packages
+  list of package names to be included in ODCS compose. Package in this case
+  refers to the "name" portion of the NVR (name-version-release) of an RPM, not
+  the Koji package name. Packages will be selected based on the Koji build tag
+  of the Koji build target used. The following command is useful in determining
+  which packages are available in a given Koji build tag:
+  ``koji list-tagged --inherit --latest TAG``
+
+pulp_repos
+  boolean to control whether or not an ODCS compose of type "pulp" should be
+  requested. If set to true, ``content_sets.yml`` must also be provided. A
+  compose will be requested for each architecture in ``content_sets.yml``.
```

Comment:
Would be nice to have some description/example about the content of `content_sets.yml`